portal-ui/src/screens/Dashboard/components/Chart/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {useTheme} from '@material-ui/core/styles';
+import {LineChart, Line, XAxis, YAxis, Label, ResponsiveContainer} from 'recharts';
+import Title from '../Title';
+
+// Generate Sales Data
+function createData(time: string, amount: number) {
+  return {time, amount};
+}
+
+const data = [
+  createData('00:00', 0),
+  createData('03:00', 300),
+  createData('06:00', 600),
+  createData('09:00', 800),
+  createData('12:00', 1500),
+  createData('15:00', 2000),
+  createData('18:00', 2400),
+  createData('21:00', 2400),
+];
+
+export default function Chart() {
+  const theme = useTheme();
+
+  return (
+    <React.Fragment>
+      <Title>Today</Title>
+      <ResponsiveContainer>
+        <LineChart
+          data={data}
+          margin={{
+            top: 16,
+            right: 16,
+            bottom: 0,
+            left: 24,
+          }}
+        >
+          <XAxis dataKey="time" stroke={theme.palette.text.secondary}/>
+          <YAxis stroke={theme.palette.text.secondary}>
+            <Label
```

Comment:
The label for the graph seems to be cut off for me, we might want to fix this:

![Screen Shot 2020-01-20 at 5 52 36 PM](https://user-images.githubusercontent.com/27314853/72768902-b927dd80-3bad-11ea-894e-9f90e124358c.png)